styleguide/components/21-atoms/logo/README.md

```diff
@@ -0,0 +1,6 @@
+# Logo
+
+## General
+
+* Clicking the logo takes you back to the homepage.
+* The logo is positioned in the most familiar placement: top left.
```

Comment:
Moet hier ook niet de business doc bij staan? bv kleurgebruik, fonts, typografie logo, geen tag lines, ...

Reply:
Mogelijks wel... ik vraag het aan Johan of hij de nodige UX informatie voor ons kan bundelen.